src/main-menu.ts

```diff
@@ -50,6 +50,9 @@ export class MainMenu extends LitElement {
     return html`
       <h1>Power Lifter</h1>
       <button @click=${this.startGame}>Play</button>
+      <button @click=${() => void this.startGame(true)}>
```

Comment:
Can we instead just read the URL params and automatically turn this on if `?quick` is supplied? Just means we don't have a button we have to remember to delete at the end before submission

Reply:
Good idea, that's a lot cleaner